### PR TITLE
[ONNX] Fix `_graph_op` type annotation

### PR DESCRIPTION
**Pull request by BowenBao**

### CODE REVIEW

**Review by justinchuby — APPROVED** [review]
thanks!

**Comment by BowenBao** [comment]
cc @garymm for merge approval

File: torch/onnx/_patch_torch.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 def _graph_op(
     g: torch._C.Graph,
     opname: str,
-    *raw_args: torch._C.Node,
+    *raw_args: torch._C.Value,
```
Comment (garymm) [inline torch/onnx/_patch_torch.py:15]: should it be a tuple of values instead? (from the star?

**Review by garymm — APPROVED** [review]

File: torch/onnx/_patch_torch.py (hunk 1 above)
Comment (justinchuby) [reply torch/onnx/_patch_torch.py:15]: We can (need to) omit the tuple: https://stackoverflow.com/questions/37031928/type-annotations-for-args-and-kwargs

**Pull request closed without merging**